seatunnel-api/src/main/java/org/apache/seatunnel/api/env/EnvCommonOptions.java

```diff
@@ -54,7 +54,7 @@ public interface EnvCommonOptions {
     Option<Long> CHECKPOINT_INTERVAL =
             Options.key("checkpoint.interval")
                     .longType()
-                    .noDefaultValue()
+                    .defaultValue(10000L)
```

Comment:
Hi, we should not change the default value. Because no `checkpoint.interval` value and have default value is different. The behavior of the two is different.